support-internationalisation/src/main/scala/com/gu/i18n/Currency.scala

```diff
@@ -40,24 +210,5 @@ object Currency {
     override def prefix: Option[String] = Some("NZ")
     override def iso: String = "NZD"
   }
-  case object SEK extends Currency {
```

Comment:
These were only used for a test in one-off contributions and are no longer needed